schema205/json_translate.py

```diff
@@ -316,10 +318,9 @@ def _load_meta_info(self, schema_section):
         if 'Root Data Group' in schema_section:
             self._schema['$ref'] = self._schema_name + '.schema.json#/definitions/' + schema_section['Root Data Group']
         # Create a dictionary of available external objects for reference
-        refs = list()
+        refs = [self._schema_name]
```

Comment:
@tanaya-bigladder this change surprised me. The translator was choosing references before the local file. I'm not sure how it was finding the right data groups before...

Reply:
This isn't the code I'm working with on my branch; maybe one of my changes hadn't been merged properly? If this works, it's fine with me, otherwise on my branch I have the order reversed: 
`        refs = list()
        if 'References' in schema_section:
            refs = schema_section['References']
        refs.append(self._schema_name)
`